Demo/src/main/java/com/paypal/android/ui/vaultcard/VaultCardViewModel.kt

```diff
@@ -155,4 +183,24 @@ class VaultCardViewModel @Inject constructor(
             securityCode = uiState.cardSecurityCode
         )
     }
+
+    fun presentAuthChallenge(activity: FragmentActivity, authChallenge: CardAuthChallenge) {
+        authChallengeState = ActionState.Loading
+
+        // change listener behavior to handle auth result
```

Comment:
^ Re-defining the listener here since the success callback is used for both 3DS and non-3DS card vaulting flows.